### DISCUSSION TITLE
Product API is not working

### DISCUSSION THREAD

**Discussion by makaloni6** [saleweaver/python-amazon-sp-api#182] [Q&A]
Hi.
I'm struggling to get correct response in Product-API for get_item_offers().

here is my code.

res_product = Products(credentials=credentials, marketplace=Marketplaces.JP)
res = res_product.get_item_offers(asin='B07QBXMY2Z', item_condition='New')

it returns following message.

sp_api.base.exceptions.SellingApiBadRequestException: [{'message': 'Missing or invalid request parameters: [ItemCondition]', 'code': 'InvalidInput'}]

I've read documentation many times, but it seems my code is correct.
How do I solve it?

Thank you.

**Comment by miogarun** [comment]
You must use ItemCondition parameter instead Item_condition

**Reply by makaloni6** [reply]
It worked! 
I didn't notice 'Item_conditioin' is wrong at all.
Thank you!
